packages/portfolio/src/adapters/dullahan-api/transformers.ts

```diff
@@ -63,7 +63,8 @@ export const toTokenActivity = (
       const [statusCode, tokenActivityData] = response
       if (statusCode !== Api.HttpStatusCode.Ok) return acc
 
-      TokenActivityResponseSchema.parse(tokenActivityData)
+      if (!TokenActivityResponseSchema.safeParse(tokenActivityData).success)
```

Comment:
I'm thinking about it, if we switch to a optimistic approach, maybe we should update all ? 🤔

Reply:
We could keep the general malformed error on the api if we wanted for the case where all of them are malformed, like

```
if (entries.length > 0 && transformedEntries.length === 0) return {tag: 'left' ...}
```

But I'm not sure that provides any value either, when I call this from the app I expect the results and I'm fine with 0 results.